### PR TITLE
Add a way to gather unmapped cells

### PR DESCRIPTION
**Pull request by freakingawesome**
This provides a way to retain unmapped cell values when fetching a strongly typed row.

### CODE REVIEW

**Comment by mrworkman** [comment]
I will try to review this week

**Comment by mrworkman** [comment]
Lol, so much for "this week". Looking at it now.

File: README.md [hunk 1]
```diff
@@ -215,6 +215,8 @@ var excel = new ExcelQueryFactory("excelFileName");
 excel.StrictMapping = StrictMappingType.Both;
 ```
 
+If you are using `None` or `ClassStrict` mapping, you can retain unmapped columns by implementing the `IContainsUnmappedCells` interface. This will put all values from the unmapped columns into a dictionary on your class named `UnmappedCells`.
```
Comment (mrworkman) [inline README.md]: We should probably include an example here by showing how to implement the interface, and then "using" one of the unmapped values.

**Comment by freakingawesome** [comment]
Thanks, @mrworkman. I've added a concrete example in the `README`.

**Comment by mrworkman** [comment]
Awesome, thanks! There was a small merge conflict introduced by another PR, but I have resolved it.

**Pull request merged**